src/Magnum/DartIntegration/ConvertShapeNode.cpp

```diff
@@ -121,6 +122,31 @@ Containers::Optional<ShapeData> convertShapeNode(dart::dynamics::ShapeNode& shap
 
             shapeData.meshes = Containers::Array<Trade::MeshData3D>(Containers::NoInit, 1);
             new(&shapeData.meshes[0]) Trade::MeshData3D{Primitives::capsule3DSolid(32, 32, 32, halfLength)};
+
+            Matrix4 rot = Matrix4::rotationX(Math::Rad<Float>(Math::Constants<Float>::piHalf()));
```

Comment:
`Rad(Constants::piHalf())` ... there are convenience typedefs for this :) Same in the two other occurences below.

Reply:
Should be fixed now!